tests/integration/project/project_validator.go

```diff
@@ -0,0 +1,70 @@
+package project
+
+import (
+	"strings"
+
+	. "github.com/onsi/gomega"
+	"github.com/openshift/odo/tests/helper"
+	"github.com/tidwall/gjson"
+)
+
+func HelpCommand() {
+	output := helper.CmdShouldPass("odo", "project", "list", "--help")
+	Expect(output).To(ContainSubstring("Specify output format, supported format: json"))
+}
+
+func GetProject() {
+	projectGetJSON := helper.CmdShouldPass("odo", "project", "get", "-o", "json")
+	getOutputJSON, err := helper.Unindented(projectGetJSON)
+	Expect(err).Should(BeNil())
+	valuesJSON := gjson.GetMany(getOutputJSON, "kind", "status.active")
+	expectedJSON := []string{"Project", "true"}
+	Expect(helper.GjsonMatcher(valuesJSON, expectedJSON)).To(Equal(true))
+}
+
+func HelpCommandProject() {
+	projectHelp := helper.CmdShouldPass("odo", "project", "-h")
+	Expect(projectHelp).To(ContainSubstring("Perform project operations"))
+}
+
+func CommandWithQFlag() {
+	projectName := helper.CmdShouldPass("odo", "project", "get", "-q")
+	Expect(projectName).Should(ContainSubstring(commonVar.Project))
+}
+
+func CommandWaitFlag() {
+	projectName := helper.RandString(6)
+	// Create the project
+	helper.CmdShouldPass("odo", "project", "create", projectName)
+
+	// Delete with --wait
+	output := helper.CmdShouldPass("odo", "project", "delete", projectName, "-f", "--wait")
+	Expect(output).To(ContainSubstring("Waiting for project to be deleted"))
+}
+
+func CommandDeleteWithJsonFlag() {
+	projectName := helper.RandString(6)
+	helper.CmdShouldPass("odo", "project", "create", projectName, "-o", "json")
```

Comment:
Why are we creating the project with json flag here? We don't seem to be testing anything specific to json output for project create.